src/core/qgsunittypes.h

```diff
@@ -557,13 +557,12 @@ class CORE_EXPORT QgsUnitTypes
     /**
      * Returns an angle formatted as a friendly string.
      * \param angle angle to format
-     * \param decimals number of decimal places to show
+     * \param decimals number of decimal places to show. By -1 it show an appropriate number of decimal places
```

Comment:
```suggestion
     * \param decimals number of decimal places to show. A value of -1 indicates that an appropriate number of decimal places should automatically be selected.
```

Reply:
thank you @nyalldawson , i committed it right now